ros/api/v1/hosts.py

```diff
@@ -57,12 +57,12 @@
 
 class IsROSConfiguredApi(Resource):
     def get(self):
-        account_number = identity(request)['identity']['account_number']
+        org_id = identity(request)['identity']['internal']['org_id']
```

Comment:
```suggestion
        org_id = org_id_from_identity_header(request)              
```

Almost everywhere ros-backend is parsing and fetch `org_id` value using nested keys fron identity header. 
Will it be good idea to encapsulate ` identity(request)['identity']['internal']['org_id']` logic into a new method [here](https://github.com/RedHatInsights/ros-backend/blob/main/ros/lib/utils.py).
After promoting `org_id` field as top level field in identity header, one needs to update code ONLY at one place.

Reply:
Yeah good idea 👍🏻

Reply:
@r14chandra  - Please go through slack conversation - https://ansible.slack.com/archives/C03B7U5AE67/p1655120034820549?thread_ts=1655111676.332289&cid=C03B7U5AE67
This change i.e org_is as top level field is already available in stage. 
Could you also use top level key to get org_id change?